tests/helpers/test_network.py

```diff
@@ -294,7 +277,7 @@ async def test_get_external_url_cloud_fallback(hass: HomeAssistant):
     )
 
     assert hass.config.external_url == "http://1.1.1.1:8123"
-    assert _async_get_external_url(hass, prefer_cloud=True) == "http://1.1.1.1:8123"
+    assert _get_external_url(hass, prefer_cloud=True) == "http://1.1.1.1:8123"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)